plugins/inputs/fireboard/README.md

```diff
@@ -0,0 +1,56 @@
+# Fireboard Input Plugin
+
+The fireboard plugin gathers the real time temperature data from fireboard
+thermometers.  In order to use this input plugin, you'll need to sign up
+to use their REST API, you can find more information on their website
+here [https://docs.fireboard.io/reference/restapi.html]
+
+### Configuration
+
+This section contains the default TOML to configure the plugin.  You can
+generate it using `telegraf --usage <plugin-name>`.
+
+```toml
+[[inputs.fireboard]]
+  ## Specify auth token for your account
+  ## https://docs.fireboard.io/reference/restapi.html#Authentication
+  # authToken = "b4bb6e6a7b6231acb9f71b304edb2274693d8849"
```

Comment:
Uncomment this option since we don't have a good default and it is required.  Don't add the URL in the config file, but lets make sure we have a link in the README.